cookbooks/cdo-freegeoip/.kitchen.yml

```diff
@@ -0,0 +1,16 @@
+---
+transport:
+  name: sftp
+driver:
```

Comment:
Good catch, it's not consistent, this `.kitchen.yml` will not work correctly as-is.

To run the integration tests for this cookbook properly on EC2 instead of Docker, you'll need to override this config with the [`.kitchen.ec2.yml`](https://github.com/code-dot-org/code-dot-org/blob/staging/cookbooks/.kitchen.ec2.yml) configuration by manually setting the KITCHEN_LOCAL_YAML environment variable before running `kitchen verify`.

I'll add a comment to this file to make this more clear.